normalized.json

```diff
@@ -187,6 +187,13 @@
     ],
     "type": "object",
     "properties": {
+        "documentType": {
```

Comment:
You might have to add `$oneOf` here as most of the time this field will be `None`